### ISSUE TITLE
[Feature Request]: Upload unobfuscated common jar on the maven

### ISSUE THREAD

**Issue by AlexiyOrlov** [FTBTeam/FTB-Mods-Issues#1436]
### Mod

FTB library

### Description

I depend on the ftb library for widgets. The common jars on the maven are obfuscated.

### Describe the solution you'd like

It would be useful if you could upload unobfuscated jars to the maven

**Comment by MichaelHillcox** [comment]
We upload a sources jar alongside the normal one, this should be enough for intelij to be able to provide good insight?

https://maven.ftb.dev/#/releases/dev/ftb/mods/ftb-library-neoforge/2101.1.7

Happy to modify things if needed, just need to fully understand why the sources jar isn't doing it's job here :P

**Comment by AlexiyOrlov** [comment]
No, I'm talking about common jar. For example, if you download source jar from [here](https://maven.ftb.dev/#/releases/dev/ftb/mods/ftb-library/2101.1.6) and look inside, it uses obfuscated names.

**Comment by MichaelHillcox** [comment]
The common and the one I linked both have sources jars? Is this not enough for you to attach sources via your ide?

**Comment by AlexiyOrlov** [comment]
Attaching sources is not a problem, they are obfuscated:
![Screenshot 2024-12-28 134316](https://github.com/user-attachments/assets/b59ba755-6dfa-42cf-b633-39511efa795e)

**Comment by MichaelHillcox** [comment]
Ah, the plot thickens, right, this seems like a gradle mess up... @desht could you look into this when you're back from your holidays

**Comment by jwright159** [comment]
Any update on this? I'm not sure how to work around it so I can access returned Minecraft classes...

**Comment by desht** [comment]
Sorry, this fell by the wayside.  How are you pulling in the FTB dependencies?  Can you paste the `dependencies` section of `build.gradle` please?
